metahtml/property/timestamp/modified.py

```diff
@@ -41,10 +42,12 @@ class Extractor(TimestampExtractor):
         ( 'niconews55.com',                 '//time[@class="date gf entry-date undo updated"]/@datetime'),
         ( 'nsk.se',                         '(//time[@itemprop="dateModified"]/@datetime)[1]'),
         ( 'ottawacitizen.com',              '//span[@class="updated-date__since"]/@date-last-update' ),
+        ( 'oa.no',                          '//time[@itemprop="dateModified"]/@datetime'),
```

Comment:
I'm noticing that created xpaths for the modified timestamps for `oa.no` and `rb.no`, but didn't create published timestamps for these domains.  Was the publication time extracted correctly automatically but the modification time not extracted?

I'm just checking because that's an unusual situation (it's normally the reverse), and I want to double check it's not a mistake.

Reply:
It was a rather strange phenomenon I encountered where the published timestamp was correct, but the modified timestamp had the date reversed. For instance, in an article where the published and modified times were the same, the modified would read `2021-05-07` for instance, when the published time would read `2021-07-05`. Of course, it is impossible that the modified time would come prior to the publishing date (especially when the article was released today)! Each time I saw this, the modified time was the inverse of the published time.

Reply:
Ahh... good catch!

That's because we stupid Americans write our dates in MM/DD/YY format, and the date parser must have been guessing to use the American standard instead of the international standard.  Anything that's not written in English should always use the international standard, so that's definitely a bug in the date parsing code that's causing it to guess incorrectly there.  I'll have to look into that.